backend/elastic/elasticUtil.js

```diff
@@ -82,15 +76,10 @@ var searchPost = async function(query_word){
         }
     })
     
-    summaryResult = summaryResult.hits.hits.map(hit => {
-        const s = {
-            data: hit._source, 
-            score: hit._score
-        }
-        return s
-    })
-
-    return {titleResult, summaryResult}
+    concatVersion = summaryResult.hits.hits.map(hit => hit._source).concat(titleResult)
```

Comment:
I would like it better if elasticUtil's search functions were directly searching from elasticsearch and you handled concats and filters on searchutil, but there really is no need right now and since we are in a rush, these are also great.